impeller/compiler/reflector.cc

```diff
@@ -652,6 +653,8 @@ std::vector<StructMember> Reflector::ReadStructMembers(
           member.vecsize == 1              //
       ) {
         // Add the type directly.
+        std::cerr << "Known Type " << maybe_known_type.value().name
```

Comment:
Debugging info?

It seems like maybe logging unknown types would be good, but probably as an FML_LOG or VALIDATION_LOG if tht's used in the compiler.